datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -142,7 +146,7 @@ CREATE VIEW dcsa_im_v3_0.event_carrier_booking_reference AS
                     'SHI' AS link_type,
                     null AS transport_call_id,
                     -- Should be shipping instruction ID when we are getting document versioning.
-                    si.id AS "document_id"
+                    si.shipping_instruction_reference AS "document_id"

```

Comment:
This should be the ID and not the reference (per comment).

Note as far as I remember, this view needs additional changes (for other document types to now use the ID rather than the reference).